### PR TITLE
added scaling text/image and reference to collection and document man…

### PR DESCRIPTION
**Pull request by fceller**
…ual entries

### CODE REVIEW

File: 1.8/arangodb/README.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ Click `Open Service` in the right upper corner to open the ArangoDB web interfac
 
 Congratulations, you now have ArangoDB running on DC/OS!
 
+To play around a bit more select `Collection` and add a new collection `example`. More information can be found [here](https://docs.arangodb.com/3.1/Manual/Administration/WebInterface/Collections.html).
+
+Click on the newly created collection. It does not have any content, yet. Click on the green plus sign, set the key to `lovedcos` and enter a JSON object. See [here](https://docs.arangodb.com/3.1/Manual/Administration/WebInterface/Document.html) for more details.
+
+## Scaling
+
+In order to scale ArangoDB switch to `Nodes`. You will see a list of coordinators and DBservers. They can be scaled independently. Add more DBservers for more data. Add more Coordinators for more computing power. See the [manual](https://docs.arangodb.com/3.1/Manual/Scalability/Architecture.html) for an in-depth explanation.
```
Comment (joerg84) [inline 1.8/arangodb/README.md]: s/to Nodes/ to the Nodes UI view.
s/coordinator/Coordinator s/DBservers/ DB Servers (Let us have it consistent with the UI).
s/manual/documentation (or have it consistent with the other links)

Both, Coordinators and DB Servers can be scaled independently. 
* Adding more DB Servers will allow you to store more data.
* Adding more Coordinators will give you more computing power while processing queries.

File: 1.8/arangodb/README.md [hunk 2]
```diff
@@ -44,6 +44,16 @@ Click `Open Service` in the right upper corner to open the ArangoDB web interfac
 
 Congratulations, you now have ArangoDB running on DC/OS!
 
+To play around a bit more select `Collection` and add a new collection `example`. More information can be found [here](https://docs.arangodb.com/3.1/Manual/Administration/WebInterface/Collections.html).
+
+Click on the newly created collection. It does not have any content, yet. Click on the green plus sign, set the key to `lovedcos` and enter a JSON object. See [here](https://docs.arangodb.com/3.1/Manual/Administration/WebInterface/Document.html) for more details.
```
Comment (joerg84) [inline 1.8/arangodb/README.md]: s/here/documentation
can we provide a sample json?

Comment (fceller) [inline 1.8/arangodb/README.md]: Done

File: 1.8/arangodb/README.md (hunk 1 above)
Comment (fceller) [inline 1.8/arangodb/README.md]: Done

File: 1.8/arangodb/README.md [hunk 3]
```diff
@@ -44,6 +44,27 @@ Click `Open Service` in the right upper corner to open the ArangoDB web interfac
 
 Congratulations, you now have ArangoDB running on DC/OS!
 
+To play around a bit more select `Collection` and add a new collection `example`. More information see the [documentation](https://docs.arangodb.com/3.1/Manual/Administration/WebInterface/Collections.html).
```
Comment (joerg84) [inline 1.8/arangodb/README.md]: For more information

**Pull request merged**